lib/src/generator/generator.dart

```diff
@@ -86,8 +86,7 @@ class Generator {
   Future<void> _updateGeneratedDir() async {
     var labels = _getLabelsFromMainArbFile();
     var locales = _orderLocales(getLocales(_arbDir));
-    var content =
-        generateL10nDartFileContent(_className, labels, locales, _otaEnabled);
+    var content = generateL10nDartFileContent(_className, labels, locales, _otaEnabled);
```

Comment:
Can we reformat this line with Dart formatter (e.g. run _"Reformat Code with dartfmt"_ context action in Android Studio) as `pub.dev` site will run a static analysis of the code and it will probably complain?